### PR TITLE
Obligatory Vox in a Corset

### PR DESCRIPTION
**Pull request by Aurorablade**
Adds Flower hairpins to the loadout.

Adds cloth footwraps (taken from polaris..might be in other places) to the loadout under racial (its meant for those of us with clawed feet..but you humans can use it I GUESS). Also in the xenodrobe locker.

Adds CORSETS by phantasmic dream. Its a suit so you can wear it over things. Blue, Red and Black.
Not in picture: Flower hairpins

:cl: Fethas
rscadd:flowerhairpins, corsets and cloth footwraps to the loadout.
/:cl:

### CODE REVIEW

**Comment by Very-Soft** [comment]
fuck yes

**Comment by Jaywings** [comment]
Now we can have lots of fancy ladies around.

**Comment by taukausanake** [comment]
So you can't wear a coat over the corset? Seems kind of counter productive in my opinion. I'd go along the line of making it both suit and uniform, or an accessory to suit, but corsets are in that weird area of clothing that kind of go both ways

**Comment by Nomzosha** [comment]
I wholeheartedly agree with taukau. I have no problems with the corsets but having them be an accessory to the suit would obviously add another huge layer of customization.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -244,3 +244,12 @@
 	desc = "For a Rustlin' tustlin' cowgirl."
 	icon_state = "cowboyboots_pink"
 	item_color = "cowboyboots_pink"
+
+/obj/item/clothing/shoes/footwraps
+ 	name = "cloth footwraps"
+ 	desc = "A roll of treated canvas used for wrapping claws or paws"
```
Comment (SamHPurp) [inline code/modules/clothing/shoes/miscellaneous.dm]: Full stop required

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -244,3 +244,12 @@
...
 	icon_state = "cowboyboots_pink"
 	item_color = "cowboyboots_pink"
+
+/obj/item/clothing/shoes/footwraps
+ 	name = "cloth footwraps"
+ 	desc = "A roll of treated canvas used for wrapping claws or paws"
+ 	icon_state = "clothwrap"
+ 	item_state = "clothwrap"
+ 	force = 0
+ 	silence_steps = 1
```
Comment (SamHPurp) [inline code/modules/clothing/shoes/miscellaneous.dm]: This is a significant balance point that you've not noted in any way on your PR description. I'll assume this wasn't an intentional omission, so please include it.

Comment (SamHPurp) [inline code/modules/clothing/shoes/miscellaneous.dm]: also , this is boolean, so `TRUE` please.

Comment (Aurorablade) [inline code/modules/clothing/shoes/miscellaneous.dm]: it just means they don't make a foot step sound...cuase its a cloth footwrap..

**Comment by Aurorablade** [comment]
@taukausanake @Nomzosha let me try it

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 2 above)
Comment (SamHPurp) [inline code/modules/clothing/shoes/miscellaneous.dm]: And not hearing peoples footsteps coming through maintenance isn't balance related?

Comment (Aurorablade) [inline code/modules/clothing/shoes/miscellaneous.dm]: you can already muffle shoes with ducttape. i mean i will add it to the changelog wheni get some second opinion on the feature in generL

**Comment by KasparoVy** [comment]
Being mindful of balance is one thing but when you have to bend logic so hard that you find yourself having to duct tape *cloth footwraps* to silence your footsteps I'm going to side with reality

**Comment by SamHPurp** [comment]
@KasparoVy and I'd agree if these weren't available loadout items, and were only mapped items / autodrobe.

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 2 above)
Comment (Fox-McCloud) [inline code/modules/clothing/shoes/miscellaneous.dm]: It's not a significant balance point though; @tigercat2000 is the one who designed this system and it was made as balance neutral as possible by making it so you could footwrap your shoes EXTREMELY easily.

**Comment by MarsM0nd** [comment]
Call me crazy, but i actually never have heard footsteps, other than the jackboots/sandals ones. Though, if they are really there, making more defined sounds might be nice. Shoes having the "normal" step sound, sandals a more wooden sound, given they are wooden sandals, and the footwraps cloth/naked feet hitting the ground, however that would actually sound. Though that probably does go beyond the scope of the PR.
Also, given that i personally never have heard the footsteps, how much of a balance thing are the stepsounds actually, or can you only hear that if you have your volume cranked up to max and then you can hear someone passing you? If it's the latter, well, that sounds like a fluff mechanic to me, such, can be fluff for footwraps that it's not there.

Completely unrelated to that, might want to consider making the footwraps produceable with cloth, given that you can also make black gloves with it, which should be harder to make than footwraps.

**Comment by taukausanake** [comment]
Foot wraps should silence stepping sound but not provide protection from stepping on things like glass. If you really want to talk balance then that's the kind to do.

I'm still a little iffy on the corsets as they are because of one thing. Whenever as a uniform, suit, or even accessory, it's going to block out another clothing sprite. As a uniform you are bottom naked (technically at least) but can cover up with a suit. As a suit you are more than likely putting the corset over a uniform which looks really weird as it's normally meant to be something you wear under clothes (or with some bottoms). You see this in the OP image with the Vox. I can't imagine anyone putting a corset over a shirt like that, let alone any jumpsuit or uniform. As an accessory it follows the same thing with it as a suit but you can still wear suits over it. It just seems like a tough thing to try and make work as the clothing system is now.

I know I said accessory before, but now I'm thinking this would work best as a lingerie and it makes me feel bad because it's like putting it lower and lower in usage

**Comment by Aurorablade** [comment]
@taukausanake cowboy shirts use the acessory system..i already change em over to use it. Corsets are an accessrory now.

**Comment by taukausanake** [comment]
@Aurorablade I mean if you're okay with having sleeves and a shirt under a corset. That's just me though

**Comment by AffectedArc07** [comment]
Question: what is a corset?

**Comment by SamHPurp** [comment]
@AffectedArc07 what was originally intended as a under garment, designed to make stomachs look thinner, hips curvier and busts bigger.

Its place in our codebase is another thing that I personally question.

**Comment by PhantasmicDream** [comment]
Nowadays people wear corsets over their clothing, having different fun fabric patterns (like fanbased things such as StarWars) 

Now they're more fun and pretty things you can have to liven up your wardrobe

**Pull request merged**